benchmarks/tpch/utils.py

```diff
@@ -0,0 +1,96 @@
+from os.path import join
+import timeit
+
+import os
+import re
+import sys
+from subprocess import run
+from linetimer import CodeTimer, linetimer
+
+FILE_TYPE = os.environ.get("FILE_TYPE", "parquet")
+SHOW_OUTPUT = bool(os.environ.get("TPCH_SHOW_OUTPUT", False))
+SCALE_FACTOR = int(os.environ.get("SCALE_FACTOR", "1"))
+LOG_TIMINGS = bool(os.environ.get("LOG_TIMINGS", True))
+
+CWD = os.path.dirname(os.path.realpath(__file__))
+DATASET_BASE_DIR = os.path.join(CWD, f"tables_scale_{SCALE_FACTOR}")
+TIMINGS_FILE = os.path.join(CWD, os.environ.get("TIMINGS_FILE", "timings.csv"))
+INCLUDE_IO = bool(os.environ.get("INCLUDE_IO", False))
+
+
+def _scan_ds(con, path: str, name):
+    path = f"{path}.{FILE_TYPE}"
+
+    q = f"create temp table {name} as select * from parquet_scan('{path}');"
+
+    con.execute(q)
+    return name
```

Comment:
Would we also be able to have runs that use `create table ...`? We could show both, but the queries using delta would be more representative of typical workloads. We'd also need to make sure that `con` is initialized with a database backed by a directory.

Reply:
added a flag to use tables instead of temp tables